lib/order.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "messager"
+
+class Order
+  attr_reader :o_items
+
+  def initialize
+    @o_items = []
+  end
+
+  def create_order_item(menu_item:, quantity:)
+    @o_items << OrderItem.new(menu_item.number, menu_item.dish, menu_item.price, quantity)
```

Comment:
Again, this inline call to .new creates a hard dependency, and will make it very difficult to properly isolate your unit tests.